sdk/storage/azure-storage-file-share-stress/src/main/java/com/azure/storage/file/share/stress/Upload.java

```diff
@@ -46,7 +49,9 @@ protected Mono<Void> runInternalAsync(Context span) {
         Flux<ByteBuffer> byteBufferFlux = new CrcInputStream(originalContent.getContentHead(), options.getSize())
             .convertStreamToByteBuffer();
         return asyncClient.uploadWithResponse(new ShareFileUploadOptions(byteBufferFlux)
-                .setParallelTransferOptions(new ParallelTransferOptions().setMaxSingleUploadSizeLong(4 * 1024 * 1024L)))
+                .setParallelTransferOptions(new ParallelTransferOptions()
```

Comment:
Can we reuse the parallel transfer options from the constructor instead of create a new options instance? I noticed us not reusing the options in a couple other test cases in the datalake stress suite (e.g. `Upload.java` and `UploadFromFile.java`)